homeassistant/components/netgear/sensor.py

```diff
@@ -244,6 +245,23 @@ class NetgearSensorEntityDescription(SensorEntityDescription):
     ),
 ]
 
+SENSOR_UTILIZATION = [
+    NetgearSensorEntityDescription(
```

Comment:
Should we disable these 2 entities by default? They tend to spam a lot and will push new updates every 30 seconds and are hardly used.

Reply:
Add `entity_registry_enabled_default=False` to both.

Reply:
These sensors are already disabled by default, see line 344/368: `_attr_entity_registry_enabled_default = False`
As part of the `NetgearRouterSensorEntity` class